blogs/rtt/round_trip_time.md

```diff
@@ -0,0 +1,109 @@
+
+In OCP ensuring efficient packet delivery is crucial for maintaining smooth
+communication between applications. However, due to various factors such
+as network congestion, misconfigured systems, or hardware limitations,
+connections may become slow, impacting overall performance.
+Round Trip Time (RTT) is the duration, measured in milliseconds, from when
+a client sends a request to when it receives a response from a server.
+Mesuring RTT is important for monitoring network health and diagnosing issues.
+
+## RTT calculation using eBPF
+
+// TODO: Explain how RTT is captured and calculed using time diffs
+// between initial SYN & ACK packets using flow_monitor hookpoint
+// Maybe also explain which are the limitations ?
+
+## Potential Use Cases
+
+Flow RTT capture from eBPF `flow_monitor` hookpoint can serve various purposes:
+
+- Network Monitoring: Gain insights into TCP handshakes, helping
+  network administrators identify unusual patterns, potential bottlenecks, or
+  performance issues.
+
+- Troubleshooting: Debug TCP-related issues by tracking latency and identifying
+  misconfigurations.
+
+## How to enable RTT
+
+To enable this feature we need to create a flow collector object with the following
+fields enabled in eBPF config section as below:
+
+```yaml
+apiVersion: flows.netobserv.io/v1beta1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  agent:
+    type: EBPF
+    ebpf:
+      sampling: 1
+      features:
+        - FlowRTT
+```
+
+Sampling 1 will ensure we capture both SYN and ACK packets to calculate RTT.
+
+## A quick tour in the UI
+
+Once the `FlowRTT` feature is enabled, the OCP console plugin automatically adapts
+to provide additional filter and show information across Netflow Traffic page views.
+
+Open your OCP Console and move to
+`Administrator view` -> `Observe` -> `Network Traffic` page as usual.
+
+A new filter, `Flow RTT` is available in the common section:
+
+![flow rtt filter](./images/flow_rtt_filter.png)
+
+It will allow you capture any flow having a RTT more than a specific time in nanoseconds.
+
+### Overview
+
+New graphs are introduced in the `advanced options` -> `manage panels` popup:
+
+![advanced options](./images/advanced_options.png)
+
+- Top X average flow RTT (donut)
+- Top X average flow RTT (lines)
+
+![rtt graphs](./images/rtt_graphs.png)
+
+These two graphs will help you to identify the slowest TCP flows and their trends
+over time. Use the filters to drill down into specific pods, namespaces or nodes.
+
+### Traffic flows
+
+The table view shows the `Flow RTT` in both column and side panel.
+
+![rtt table](./images/rtt_table.png)
+
+### Topology
+
+Last but not least, the topology view displays average `RTT` latency on edges.
+Clicking on a node or an edge will allow you to see per direction metrics and
+the related graph.
+
+![rtt topology](./images/rtt_topology.png)
+
+### Future improvments
+
+Here is a non exhaustive list of future improvments coming for a full featured
```

Comment:
Misspelling here and in the title (line 95):
`improvements`

Reply:
Done https://github.com/netobserv/documents/pull/54/commits/2a3dde7605b7231dc87de1f71eb93cb2d7b66266